### PR TITLE
feat(data-import): uploaded file is converted to json by rasa endpoint

### PR DESCRIPTION
**Pull request by Palashx**
I made sure that the data imports test passes

### CODE REVIEW

File: botfront/imports/ui/components/nlu/import-export/DataImport.jsx [hunk 1]
```diff
@@ -32,12 +32,17 @@ export default class DataImport extends React.Component {
     }
 
     onDrop = (files) => {
+        const { model, instanceHost } = this.props;
         files.forEach((file) => {
             const reader = new FileReader();
-            reader.onload = () => this.setState({ values: reader.result });
+            reader.onload = () => {
+                Meteor.call('nlu.convertToJson', reader.result, model.language, 'json', instanceHost, wrapMeteorCallback((err, result) => {
+                    if (err) throw new Meteor.Error('Error in converting data', err);
```
Comment (MatthieuJnon) [inline botfront/imports/ui/components/nlu/import-export/DataImport.jsx]: don't throw Meteor.error on front end and only update state if !err

File: botfront/imports/ui/components/nlu/import-export/DataImport.jsx [hunk 2]
```diff
@@ -32,12 +32,17 @@ export default class DataImport extends React.Component {
     }
 
     onDrop = (files) => {
+        const { model, instanceHost } = this.props;
         files.forEach((file) => {
             const reader = new FileReader();
-            reader.onload = () => this.setState({ values: reader.result });
+            reader.onload = () => {
```
Comment (MatthieuJnon) [inline botfront/imports/ui/components/nlu/import-export/DataImport.jsx]: Don't convert to json if its already a json file

Comment (Palashx) [reply botfront/imports/ui/components/nlu/import-export/DataImport.jsx]: ok will do it

File: botfront/imports/ui/components/nlu/import-export/DataImport.jsx (hunk 1 above)
Comment (Palashx) [reply botfront/imports/ui/components/nlu/import-export/DataImport.jsx]: ok got it, will fix it

**Pull request merged**